libpod/container_internal.go

```diff
@@ -1424,9 +1421,6 @@ func (c *Container) cleanupStorage() error {
 		if errors.Cause(err) == storage.ErrNotAContainer || errors.Cause(err) == storage.ErrContainerUnknown || errors.Cause(err) == storage.ErrLayerNotMounted {
 			logrus.Errorf("Storage for container %s has been removed", c.ID())
 		} else {
-			if cleanupErr != nil {
-				logrus.Errorf("Error cleaning up container %s storage: %v", c.ID(), cleanupErr)
-			}
```

Comment:
It seems strange to log when we ignore the error and ignore the error here.

Reply:
I think we still need this bit, otherwise we lose the contents of cleanupErr

Reply:
Well if you read the code, the cleanupErr has never been set.  This is the first time.